Monopoly/Assets/Scripts/Game.cs

```diff
@@ -295,4 +388,25 @@ void SetJail()
         currentPlayer.GoToJail();
         infoPopup.ShowMessage("Idziesz do więzienia", "Będziesz pauzował 3 tury");
     }
-}
+
+    void HandlePlayerBancrupcy(Player debtor, Player creditor)
+    {
+        foreach(Property property in debtor.ownedProperties)
+        {
+            creditor.ownedProperties.Add(property);
+            property.ChangeOwner(creditor.GetId());
+        }
+        players.Remove(debtor);
+        numberOfPlayers--;
+        playerNames.Remove(debtor.playerName);
+        debtor.Disable();
+        moneyManager.PlayerBancrupcy(debtor);
```

Comment:
moneyManager.PlayerBancrupcy robi Remove na liście players. A kilka linii wyżej jest to samo, nie możemy dwa razy usuwać jednego gracza, bo sypnie jakimś nullem. Moim zdaniem może to być nie w klasie moneyManager, tylko tutaj w Game.cs, tak jak kilka linii wyżej jest teraz.

Reply:
To są zupełnie dwie inne listy